### PR TITLE
Log shipping/cert technical note

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

Added a technical note on replacing the logzio cert locally

----

## Pages to review

- https://deploy-preview-501--logz-docs.netlify.app/technical-notes/chain-of-trust/

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information: Support, CSEs

### CODE REVIEW

File: _source/logzio_collections/_technical-notes/chain-of-trust.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+a few days early. This new chain won’t expire until 2038.
+
+This means you’ll need to update any of your data shipping
+configurations that make use of our certificate chain.
+Configurations need to be updated before May 28
+to avoid disruptions in sending your data to Logz.io.
+
+## What this affects
+
+Many of the shipping methods covered in the Logz.io docs are affected,
```
Comment (ofer-velich) [inline _source/logzio_collections/_technical-notes/chain-of-trust.md:28]: maybe say "some effected but not all"?

File: _source/logzio_collections/_technical-notes/chain-of-trust.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+##### Test the new certificate
+
+Starting Monday, May 18,
+we’ll open up test listeners
+so you can confirm the new configurations will work
+after we switch to the new chain of trust.
+
+Test your new configuration
+by temporarily pointing your shipper to a testing URL for your region
```
Comment (ofer-velich) [inline _source/logzio_collections/_technical-notes/chain-of-trust.md:140]: suggestion 
```
by temporarily pointing a local copy of the shipper to a testing URL for your region
```
they dont need to point all of their shippers, but only to test it on one local **or** remote server

File: _source/logzio_collections/_technical-notes/chain-of-trust.md (hunk 1 above)
Comment (imnotashrimp) [reply _source/logzio_collections/_technical-notes/chain-of-trust.md:28]: good point. will be in the next push before merging.

File: _source/logzio_collections/_technical-notes/chain-of-trust.md (hunk 2 above)
Comment (imnotashrimp) [reply _source/logzio_collections/_technical-notes/chain-of-trust.md:140]: I'm going to leave it — I'd rather they made their own decisions about what to test.

**Pull request merged**